website/app/components/doc/font-helpers-list/index.hbs

```diff
@@ -5,7 +5,8 @@
 
 <ul class="doc-font-helpers-list" role="list">
   {{#each @items as |item|}}
-    <li class="doc-font-helpers-list__item">
+    {{! Add role="listitem" because the class sets display: contents; }}
```

Comment:
```suggestion
    {{! role="listitem" is needed here because the class sets `display: contents` and some browsers and assistive technologies will ignore the implied role }}
```